### PR TITLE
Rename bitwarden rs to vaultwarden

### LINKED ISSUE #296: Update Bitwarden_rs refernces to Vaultwarden
The Bitwarden_rs project has been renamed to Vaultwarden. References and links to said project need to be updated.

### LINKED ISSUE #298: Vaultwarden alpine is not available for arm64
Vaultwarden alpine is not available for arm64, for best compatibility use the 'normal' latest tag.

Alternatively offer a Vaultwarden and a Vaultwarden-alpine Rockon solution.

### PR DESCRIPTION
**Pull request by RlndVt**
Fixes #296.

E:
Expanded to also fix #298 .

### CODE REVIEW

**Comment by phillxnet** [comment]
@RlndVt & @Hooverdan96 Well done on resolving the related arm issue re #298.

Are we now just awaiting @Hooverdan96 's suggestions in that issue re:

https://github.com/rockstor/rockon-registry/issues/298#issuecomment-981996589

> So, I think, once the description in #297 is updated to include the available architecture language, then this could likely be closed.

The only architectures we are interested in are x86_64 & ARM64 really. The v7 arms are way under-powered for Rockstor and btrfs.

@RlndVt do you mind updating this pr's original text to list it's (welcome in this case) scope creep? I.e. it looks like it also fixes the arm compatibility/description (with edits)  issue so another "Fixes" line could be added for example. All helps with assessing the state as this one is getting a little confusing with it's 3 associated issues :). All good.

Cheers. Almost there on this one by the looks of it. And thanks again for all you efforts. It will be good to get this one done and dusted.

**Comment by RlndVt** [comment]
Squashed the root.json commits as they shouldn´t be considered separate from the rest.

File: vaultwarden.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "Vaultwarden": {
+        "description": "Alternative implementation of the Bitwarden password manager server API written in Rust and on Alpine (speed, size). <p>Based on the following docker image: <a href='https://hub.docker.com/r/vaultwarden/server' target='_blank'>https://hub.docker.com/r/vaultwarden/server</a></p>.",
```
Comment (FroggyFlox) [inline vaultwarden.json:3]: Add architecture compatibility:
```suggestion
        "description": "Alternative implementation of the Bitwarden password manager server API written in Rust and on Alpine (speed, size). <p>Based on the following docker image: <a href='https://hub.docker.com/r/vaultwarden/server' target='_blank'>https://hub.docker.com/r/vaultwarden/server</a> compatible with amd64 and arm64 architectures.</p>",
```

**Comment by FroggyFlox** [comment]
Hi @RlndVt ,

First, please accept my apologies for taking so long to have a proper look at this. I've tested it on amd64 and it works as intended. I'll try to test it on an arm64 machine later today to make sure that works as well but in the meantime I thought I would make a suggestion to add this multi-architecture compatibility. Feel free to commit that suggestion when you get a chance.

Thanks again for submitting this PR and keeping this Rock-on up-to-date!

**Comment by FroggyFlox** [comment]
@RlndVt ,

I just tested on both amd64 and arm64 architectures and it works like a charm. I created a self-signed certificate in the share to be used as "vaultwarden certificates":
```
openssl req -newkey rsa:2048 -nodes -keyout key.pem -x509 -days 365 -out certs.pem
```

... then installed the Rock-on and it all works as intended:
- installs without error
-  webUI is accessible
- restarts normally
- restarts automatically and without error after a reboot
- can be uninstalled and re-installed without a problem


I think we are ready to merge this in as soon as we have the little edits related to the explicit mention of the multi-arch compatibility in the Rock-on `"description"` field.

Thanks a lot!

**Review by FroggyFlox — APPROVED** [review]

**Comment by FroggyFlox** [comment]
Thanks for the additional edits, @RlndVt !

I'll merge this now.

**Comment by FroggyFlox** [comment]
This has now been published. Thanks again @RlndVt !

**Comment by FroggyFlox** [comment]
This has now been published. Thanks again @RlndVt !

**Pull request merged**